modules/installing-gcp-cluster-creation.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+// * installing-gcp-user-defined-labels-and-tags.adoc
+
+:_content-type: PROCEDURE
+[id="installing-gcp-cluster-creation_{context}"]
+= Cluster creation phase 
+
+After you successfully install an add-on service on your {product-title} cluster, you can access the service by using the GCP.
+
+.Prerequisites
+
+* You have successfully installed a service on your {product-title} cluster.
+
+* The installation program requires that a service account includes a `TagUser` role, so that the program can create the {product-title} cluster with defined tags at both organization and project levels.
+
+.Procedure
+
+[NOTE]
+====
+`Labels` and `tags` are defined during the `install-config.yaml` creation phase, and cannot be modified or updated with new `labels` and `tags` after cluster creation.
+====
+
+:FeatureName: Support for user-defined `labels` and `tags` for GCP
+include::snippets/technology-preview.adoc[]
+
+. Use `.platform.gcp.userLabels` and `platform.gcp.userTags` fields to define the list of desired labels and tags in `install-config.yaml` file.
```

Comment:
It should mention about updating the file instead of using the fields

Reply:
Changed